docs/spec/spec.md

```diff
@@ -0,0 +1,310 @@
+
+\*: Required
+
+1: Kubernetes type.
+
+#### Metadata
+
+##### Owner References
+
+- Owns EventTypes.
+
+#### Status
+
+| Field       | Type                      | Description                                                          | Limitations                                                                    |
+| ----------- | ------------------------- | -------------------------------------------------------------------- | ------------------------------------------------------------------------------ |
+| provisioned | []ProvisionedObjectStatus | Status of creation or adoption of each EventType and errors therein. | It is expected that a provisioner list all produced EventTypes, if applicable. |
+
+#### Events
+
+- Source created
+- Source deleted
+- Event types installed
+
+### Life Cycle
+
+| Action | Reactions                                                                       | Limitations                                                                                              |
+| ------ | ------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| Create | Creates and owns EventTypes produced, or adds Owner ref to existing EventTypes. | Verifies Json Schema provided by existing EventTypes; Not allowed to edit EventType if previously Owned; |
+| Update | Synchronizes EventTypes.                                                        |                                                                                                          |
+| Delete | Removes Owner ref from EventTypes.                                              |                                                                                                          |
+
+---
+
+## kind: Subscription
+
+### group: eventing.knative.dev/v1alpha1
+
+_Describes a linkage between a Subscribable and a Targetable and/or Sinkable._
+
+### Object Schema
+
+#### Spec
+
+| Field              | Type         | Description                                                                  | Limitations        |
+| ------------------ | ------------ | ---------------------------------------------------------------------------- | ------------------ |
+| from\*             | ObjectRef    | The originating Channel for the link.                                        | Must be a Channel. |
+| call<sup>1</sup>   | EndpointSpec | Optional processing on the event. The result of call will be sent to result. |                    |
+| result<sup>1</sup> | ObjectRef    | The continuation Channel for the link.                                       | Must be a Channel. |
+
+\*: Required
+
+1: At Least One(call, result)
+
+#### Metadata
+
+##### Owner References
+
+- If a resource controller created this Subscription: Owned by the originating
+  resource.
+
+##### Conditions
+
+- **Ready.**
+- **FromReady.**
```

Comment:
Configured?

Reply:
I matched what is implemented. I think a followup from the implementor is required.

Reply:
continue: https://github.com/knative/eventing/issues/535

in a followup